frontend/mobilityai/app/src/main/java/com/ai/mobility/mobilityai/MetaMotionService.java

```diff
@@ -0,0 +1,224 @@
+import bolts.Task;
+
+public class MetaMotionService {
+    //Metawear components
+    private MetaWearBoard board;
+    private BtleService.LocalBinder serviceBinder;
+    private Led led;
+    private Accelerometer accelerometer;
+    private GyroBmi160 gyroscope;
+    private Logging logging;
+    private String macAddr;
+
+    //Testing
+    private final String MW_MAC_ADDRESS= "D1:87:11:D8:F3:C0";
+    private static final String TAG = "MobilityAI";
+
+    public MetaMotionService(BtleService.LocalBinder serviceBinder) {
+        //Connect to boards
+        this.serviceBinder = serviceBinder;
+    }
+
+    //Connect
+    //Disconnect
+    //Copy Data + clear existing data
+    //  helper - format data into csv/json
+    //Start Sensor
+    //Stop Sensor
+    //Count footsteps
+
+
+    /**
+     * Connect to board and populate sensor component variables
+     * @param macAddr MAC address of device to connect
+     * @param remoteDevice BluetoothDevice to connect
+     */
+    public void connect(String macAddr, BluetoothDevice remoteDevice) {
+        board = serviceBinder.getMetaWearBoard(remoteDevice);
+        this.macAddr = macAddr;
+
+        board.connectAsync().continueWithTask(task -> {
+            if (task.isFaulted()) {
+                Log.i("MobilityAI", "Failed to configure app", task.getError());
+            } else {
+
+                Log.i("MobilityAI", "App Configured, connected: " + this.macAddr);
+
+                //Retrieve Modules
+                led = board.getModule(Led.class);
+                accelerometer = board.getModule(Accelerometer.class);
+                gyroscope = board.getModule(GyroBmi160.class);
+            }
+
+            return null;
+        });
+
+//            @Override
+//            public Void then(Task<Void> task) throws Exception {
+//
+//
+////                    //Configure accelerometer to set sampling rate to 25Hz
+////                    accelerometer.configure()
+////                            .odr(25f)       // Set sampling frequency to 25Hz, or closest valid ODR
+////                            .commit();
+////
+////                    //Add route to log accelerometer data
+////                    return accelerometer.acceleration().addRouteAsync(new RouteBuilder() {
+////                        @Override
+////                        public void configure(RouteComponent source) {
+////                            source.stream(new Subscriber() {
+////                                @Override
+////                                public void apply(Data data, Object... env) {
+////                                    Log.i(TAG, "Accelerometer: " + data.value(Acceleration.class).toString());
+////                                }
+////                            });
+////                        }
+////                    });
+//
+//                }
+//            }
+//        });
+    }
+
+    public void disconnect() {
+
+    }
+
+    public void setBatteryLevel(int id) {
```

Comment:
Don't seem to use this, is it necessary?

Reply:
Same with `disconnect()`

Reply:
Those are placeholders for now, I'm going to copy over some functions from the other files later to remove some of the code duplication